packages/openid4vc/src/openid4vc-issuer/OpenId4VcIssuerModuleConfig.ts

```diff
@@ -226,12 +242,31 @@ export class OpenId4VcIssuerModuleConfig {
   /**
    * The time after which an access token will expire.
    *
-   * @default 360 (5 minutes)
+   * @default 180 (3 minutes)
    */
   public get accessTokenExpiresInSeconds(): number {
     return this.options.accessTokenExpiresInSeconds ?? DEFAULT_TOKEN_EXPIRES_IN
   }
 
+  /**
+   * The time after which a refresh token will expire.
+   *
+   * @default 7776000 (90 days)
+   */
+  public get refreshTokenExpiresInSeconds(): number {
+    return this.options.refreshTokenExpiresInSeconds ?? DEFAULT_REFRESH_TOKEN_EXPIRES_IN
+  }
+
+  /**
+   * Whether to generate refresh tokens for issuance sessions. This can be useful if
+   * you're planning on deferring the issuance of credentials.
+   *
+   * @default false
+   */
+  public get generateRefreshTokens(): boolean {
```

Comment:
I think it's useful to be able to set this on a per-issuance session basis as well, is that possible? At least in Paradym I see us making deferred issuance a thing you configure on a credential template, and thus it's not a global setting

Reply:
I agree